extensions/oplogPopulator/OplogPopulatorTask.js

```diff
@@ -1,34 +1,53 @@
 'use strict'; // eslint-disable-line
 
 const werelogs = require('werelogs');
+const schedule = require('node-schedule');
 const config = require('../../lib/Config');
 const OplogPopulator = require('./OplogPopulator');
 
 const logger = new werelogs.Logger('Backbeat:OplogPopulator:task');
 werelogs.configure({ level: config.log.logLevel,
     dump: config.log.dumpLevel });
 
+const qpConfig = config.queuePopulator;
 const mongoConfig = config.queuePopulator.mongo;
 const oplogPopulatorConfig = config.extensions.oplogPopulator;
+const activeExtensions = Object.keys(config.extensions);
 
 const oplogPopulator = new OplogPopulator({
     config: oplogPopulatorConfig,
     mongoConfig,
+    activeExtensions,
     logger,
 });
 
 (async () => {
     try {
         await oplogPopulator.setup();
+        schedule.scheduleJob(qpConfig.cronRule, async () => {
+            try {
+                // Update each connector's config
+                await Promise.all(oplogPopulatorConfig.connectors.map(connConf =>
```

Comment:
is this tacking only the i think we should have a single connector for now?
dispatching onto multiple connectors is next ticket...